tests/manage/monitoring/conftest.py

```diff
@@ -277,3 +277,59 @@ def stop_mon():
         assert check_old_mons_deleted, msg
 
     return measured_op
+
+
+@pytest.fixture
+def workload_stop_ceph_osd(measurement_dir):
```

Comment:
name should be measure_ceph_stop_osd for better representing what it does.

Reply:
Good idea! I will change it next PR so it is changed for all alerting tests at once. Fixtures for all alerting tests are currently named this way but this PR is only about one test case.